de.manuel_kehl.go-for-it.appdata.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Patrik Nilsson -->
+<component type="desktop-application">
+  <id>de.manuel_kehl.go-for-it.desktop</id>
+  <name>Go For It!</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <summary>A stylish to-do list with built-in productivity timer</summary>
+  <developer_name>Manuel Kehl</developer_name>
+  <description>
+    <p>
+      Go For It! is a simple and stylish productivity app, featuring a to-do
+      list, merged with a timer that keeps your focus on the current task.
+    </p>
+    <p>
+      To-do lists are stored in the Todo.txt format. This simplifies
+      synchronization with mobile devices and makes it possible to edit tasks
+      using other front-ends, like my Todo.txt Kupfer Plugin.
+    </p>
+    <p>
+      If you already use Todo.txt, beware of the fact, that Go For It!
+      automatically archives completed tasks to the done list!
+    </p>
+  </description>
+  <url type="homepage">http://manuel-kehl.de/projects/go-for-it/</url>
+  <url type="bugtracker">https://github.com/mank319/Go-For-It/issues</url>
+  <url type="help">https://github.com/mank319/Go-For-It/blob/master/README.md</url>
+  <url type="translate">https://translations.launchpad.net/go-for-it</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/AsavarTzeth/flathub/de.manuel_kehl.go-for-it/images/screenshot-01.png</image>
```

Comment:
I wonder if we shouldn't have a more permanent home for these screenshots; I also see that you have the screenshot files committed to git, which should not be required.

Reply:
I find this sort of funny. I only did this because you already have applications on flathub that do this, like [io.mgba.mGBA](https://github.com/flathub/io.mgba.mGBA).

In any case, this was only meant to be temporary. I suppose I could upload them to some service. I vaguely remember reading about some services meant for this in the AppStream documentation.